src/api/c/graphics_common.cpp

```diff
@@ -174,7 +174,7 @@ namespace graphics
 {
 ForgeManager& ForgeManager::getInstance()
 {
-    static ForgeManager my_instance;
+    thread_local ForgeManager my_instance;
```

Comment:
I didn't see any mutex locks around the calls for the ForgeManager methods. I thought it was safer to make this thread local.